langlib/lang.string/src/main/ballerina/src/lang.string/string.bal

```diff
@@ -41,48 +48,105 @@ public function iterator(string str) returns abstract object {
 }
 
 # Concatenate all the `strs`. Empty string if empty.
+#
+# + strs - strings to concat
+# + return - concatanated string
 public function concat(string... strs) returns string = external;
 
+# Returns the unicode codepoint at index `i`.
+#
+# + str - the string
+# + i - code point index
+# + return - code point
 public function getCodePoint(string str, int i) returns int = external;
 
 // todo: endIndx should be a defaultable param: int endIndx = str.length()
+# Returns a string that is a substring of this string.
+#
+# + str - source string.
+# + startIndex - the beginning index, inclusive.
+# + endIndex - the ending index, exclusive.
```

Comment:
```suggestion
# + endIndex - the ending index, exclusive.
```
Why do we need to say "inclusive" here? Instead, shall we say "this is optional"?